presto-orc/src/main/java/com/facebook/presto/orc/OrcRecordReaderOptions.java

```diff
@@ -23,18 +25,25 @@
     private final DataSize tinyStripeThreshold;
     private final DataSize maxBlockSize;
     private final boolean mapNullKeysEnabled;
+    private final TimeUnit timestampTimeUnit;
 
     public OrcRecordReaderOptions(OrcReaderOptions options)
     {
-        this(options.getMaxMergeDistance(), options.getTinyStripeThreshold(), options.getMaxBlockSize(), options.mapNullKeysEnabled());
+        this(options.getMaxMergeDistance(), options.getTinyStripeThreshold(), options.getMaxBlockSize(), options.mapNullKeysEnabled(), options.getTimestampTimeUnit());
     }
 
-    public OrcRecordReaderOptions(DataSize maxMergeDistance, DataSize tinyStripeThreshold, DataSize maxBlockSize, boolean mapNullKeysEnabled)
+    public OrcRecordReaderOptions(
+            DataSize maxMergeDistance,
+            DataSize tinyStripeThreshold,
+            DataSize maxBlockSize,
+            boolean mapNullKeysEnabled,
+            TimeUnit timestampTimeUnit)
     {
         this.maxMergeDistance = requireNonNull(maxMergeDistance, "maxMergeDistance is null");
         this.maxBlockSize = requireNonNull(maxBlockSize, "maxBlockSize is null");
         this.tinyStripeThreshold = requireNonNull(tinyStripeThreshold, "tinyStripeThreshold is null");
         this.mapNullKeysEnabled = mapNullKeysEnabled;
+        this.timestampTimeUnit = requireNonNull(timestampTimeUnit);
```

Comment:
nit: message is missing 

`this.timestampTimeUnit = requireNonNull(timestampTimeUnit, "timestampTimeUnit is null");`

Reply:
changed to boolean